reference/req/req.host.md

```diff
@@ -1,10 +1,15 @@
 # req.host
-The hostname of this request, without the port number, as specified by its "Host" header.
 
+> ##### **This method is deprecated and will likely be removed or changed in an upcoming release.**
+> Instead, use: [req.hostname](https://github.com/balderdashy/sails-docs/edit/master/reference/req/req.hostname.md)
```

Comment:
@streleck Would you change this to the sailsjs.com URL?  It'll be automatically adapted for `next.` and other future docs subdomains